spec/mongoid/copyable_spec.rb

```diff
@@ -54,6 +54,30 @@
         end
       end
 
+      context "when using store_as" do
+        context "and dynamic attributes are not set" do
```

Comment:
can you add space here and in the next line, to keep the file style consistent ?